kommunicate/src/main/java/io/kommunicate/services/KmHttpClient.java

```diff
@@ -96,7 +100,7 @@ public String postData(String urlString, String contentType, String accept, Stri
         return null;
     }
 
-    public String getResponseWithException(String urlString, String contentType, String accept, Map<String, String> headers) throws Exception {
+    public String getResponseWithException(String urlString, String contentType, String accept, Map<String, String> headers) throws Throwable {
```

Comment:
any specific reason for changing Exception with Throwable ?

Reply:
done, please check